### PR TITLE
Windows preview release

### PR DESCRIPTION
**Pull request by loganhz**

### CODE REVIEW

**Comment by deniseschannon** [comment]
Should we completely remove the directions on how to use Windows in earlier releases?

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 1]
```diff
@@ -44,73 +45,67 @@ To begin provisioning a custom cluster with Windows support, prepare your host s
...
+>- If you are using AWS, you can choose *Microsoft Windows Server 2019 Base with Containers* as Amazon Machine Image (AMI)
+>- If you are using GCE, you can choose *Windows Server 2019 Datacenter for Containers* as OS image
 
 ### Requirements
 
 - You can view node requirements for Linux and Windows nodes in the [installation section]({{< baseurl >}}/rancher/v2.x/en/installation/requirements/).
-- All nodes in a virtualization cluster or a bare metal cluster must be connected using a layer 2 network.
-- To support [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/), your cluster must include at least one Linux node dedicated to the worker role.
+- For **Host Gateway (L2bridge)** networking, it's best to use the same Layer 2 network for all nodes. Otherwise, you need to configure the route rules for them.
+- For **VXLAN (Overlay)** netwokring, you must confirm the Windows Server 2019 host with [KB4489899](https://support.microsoft.com/en-us/help/4489899) hotfix. Most of Cloud-hosted VMs already have this hotfix.
```
Comment (dnoland1) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md]: Typo - netwokring should be networking

**Comment by loganhz** [comment]
@deniseschannon Can we move the previous docs for windows to a separated page?

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 1 above)
Comment (loganhz) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md]: updated. Thanks!

**Comment by deniseschannon** [comment]
@loganhz yes, let's do what we did with pipelines 1.0.

**Pull request merged**